packages/telemetry/src/slog-to-otel.js

```diff
@@ -561,6 +561,11 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
         spans.end(`begin-block`);
         break;
       }
+      case 'cosmic-swingset-commit-block': {
+        spans.top().setAttributes(cleanAttrs(slogAttrs));
+        spans.pop(`block ${slogAttrs.blockHeight}`);
```

Comment:
I forgot to highlight, but I believe this will not dramatically change the `block ${blockHeight}` span. Previously, the span would be ended with the time of the last/previous named pop, which is `end-block` finish. This effectively will include the time between end-block-finish and commit-block in the block time where it was previously excluded.

Furthermore the `setAttributes` call assumes `top()` is the block span, but that's only enforced in the subsequent `pop` call.  Technically if the layering of spans change in the future but not everything is updated accordingly, we may end up with adding attributes to another span, and only "throw" on the `pop` which simply results in console error output since #5578

Reply:
As soon as I figure out `slog-to-otel.js`, I'm planning to add a bunch of spans (about three per syscall), but they should all be children of the crank/delivery span, not the block span, so I don't think that will change the layering you describe.

Reply:
Discussed offline, will be handled in a follow-up PR.